csrc/device_lower/analysis/tensor_memory.cpp

```diff
@@ -7,12 +7,59 @@
 // clang-format on
 
 #include <device_lower/analysis/tensor_memory.h>
+#include <device_lower/lower2device.h>
 #include <fusion.h>
 #include <ir/all_nodes.h>
 #include <type.h>
 
+#include <utility>
+#include <vector>
```

Comment:
Nit: while we're at it :-)
```suggestion
#include <unordered_set>
#include <utility>
#include <vector>
```

Reply:
fixed